.readthedocs.yaml

```diff
@@ -5,10 +5,10 @@
 # Required
 version: 2
 
-# Include the framework submodule in the build
+# Include all submodules in the build
 submodules:
-  include:
-  - framework
+  include: all
```

Comment:
That's quite equivalent I'd say. It is more align with how we do throughout the CI like this I think. Otherwise with that the framework submodule in tf-psa-crypto is checked out as well.